moveit_ros/planning/rdf_loader/src/synchronized_string_parameter.cpp

```diff
@@ -119,7 +119,9 @@ bool SynchronizedStringParameter::shouldPublish()
 
 bool SynchronizedStringParameter::waitForMessage(const rclcpp::Duration timeout)
 {
-  string_subscriber_ = node_->create_subscription<std_msgs::msg::String>(
+  std::string const nd_name = std::string(node_->get_name()).append("_ssp_").append(name_);
```

Comment:
Use auto, you don't need to specify that this is a string twice on the same line.

```suggestion
  auto const nd_name = std::string(node_->get_name()).append("_ssp_").append(name_);
```